libdirections-hybrid/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     api(project(':libnavigation-base'))
     implementation(project(":libdirections-onboard"))
     implementation(project(":libdirections-offboard"))
+    implementation project(':libnavigation-util')
```

Comment:
Let's add the dependency only when the module will actually need any of the tools from `libnavigation-utils`. Same for all other modules.

Reply:
good catch. done 👍